modules/onboard-app/outputs.tf

```diff
@@ -0,0 +1,12 @@
+//output "pipeline_id" {
```

Comment:
Everything in this file is commented out.... can delete?

Reply:
yup deleting.